portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,69 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 600px;
 }
-
-#greeting-container {
+#image {
+    margin-left: auto;
+    margin-right:auto;
+    margin-top: auto;
+    width: 128px;
+}
+#fact-container {
   margin-top: 20px;
 }
+
+body {
+    background-color: lightblue;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, 
+    Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+
+}h1 {
+    text-align: center;
+}
+
+h2 {
+    text-align: center;
+}
+
+label.logo {
+     color: white;
+     font-size: 55px;
+     font-weight: bold;
+     line-height: 75px;
+     padding: 100px;
+     
+ }
+
+nav {
+    background-color: royalblue;
+    height: 70px;
+    width: 100%
+}
+
+nav ul{
+    float: right;
+    margin-right: 20px;    
+}
+
+nav ul li {
+display: inline-block;
+line-height: 80px;
+margin: 8px;
+}
+
+nav ul li a{
+    color: white;
+    font-size: 20px;
+    text-transform: uppercase;
+}
+
+
+/*Navigation Bar Creation*/
+* {
+    padding: 0;
```

Comment:
Alphabetize these CSS properties. The rest of your file looks wonderfully alphabetized, though!